lib/phoenix/controller.ex

```diff
@@ -159,16 +162,20 @@ defmodule Phoenix.Controller do
   connection-agnostic and typically invoked from your views.
   """
   defmacro __using__(opts) do
-    quote bind_quoted: [opts: opts] do
+    {put_default_views, opts} = Keyword.pop(opts, :put_default_views, true)
```

Comment:
If you do this here, you will force the options to be given at compile time and something like this will break:

    opts = [put_default_views: false]
    use Phoenix.Controller, opts

It is generally a no-no to process options outside of quote. :)